cmd/fyne/internal/util/file.go

```diff
@@ -39,6 +39,31 @@ func EnsureSubDir(parent, name string) string {
 	return path
 }
 
+// EnsureAbsPath returns the absolute path of the given file if it is not already absolute
+func EnsureAbsPath(path string) string {
+	if !filepath.IsAbs(path) {
+		abs, err := filepath.Abs(path)
+		if err != nil {
+			fyne.LogError("Failed to find absolute path", err)
+		} else {
+			return abs
+		}
+	}
+	return path
+}
+
+// MakePathRelative joins root with path if path is not absolute and exists in root
+// both root and path must non-empty, otherwise path will be returned
+func MakePathRelative(root, path string) string {
```

Comment:
This name seems misleading - the path returned is absolute.
Perhaps you meant `MakePathRelativeTo`?